### DISCUSSION TITLE
List apps installed from Aurora Store

### DISCUSSION THREAD

**Discussion by Jean-BaptisteC** [Exodus-Privacy/exodus-android-app#151] [Ideas]
Aurora Store get apps from Play Store,
Actually in Exodus app, we only show apps installed from Play Store and Fdroid.
You can support Aurora Store on your app easily, with add Aurora Store filter to show apps installled on smartphone.
And on reports on app, you can show source: Google like on apps installed from Play Store.

**Comment by theimpulson** [comment]
We can now list all of the apps installed on the device irrespective of the store with the new `QUERY_ALL_PACKAGES` permission we added recently, irrespective of the store used to install them. The logic right now is limited to only showing certain apps with certain flags which results in showing less apps.

**Comment by yoshimo** [comment]
There are other apps that install apps. Amazon, aurora droid,  droid-if, vendor stores from xiaomi or Samsung.
If we can avoid showing only white listed sources, it would be great.

**Comment by Jean-BaptisteC** [comment]
We filter list app (on source installation from Play Store or Fdroid) because on your website, we show only reports on apps installed from Play Store or Fdroid.
Show other apps without can show trackers is useless. (Permissions can be find on settings)

We add support for Aurora Store because this store is fork of Play Store.
Of course, if we add more stores on website, we add this stores on Android app

**Reply by yoshimo** [reply]
Keep in mind that f-droid has multiple front ends though as anyone can write them.
There is also the possibility to download Google packages from websites like apk pure and either side load them with a manual installation or with an installer app.

**Reply by Jean-BaptisteC** [reply]
We have add support for only Aurora Store (Google and Fdroid has already present inapp).
Apps install manually by user is show with source "User"
